src/main/java/fr/insalyon/creatis/gasw/parser/GaswParser.java

```diff
@@ -112,7 +116,7 @@ public void parse(String descriptorFileName) throws GaswException {
 
     @Override
     public void startElement(String uri, String localName, String qName, Attributes attributes) throws SAXException {
-
+    	logger.info("startElement: " + uri+" : "+localName+" : "+qName+" : "+attributes.toString());
```

Comment:
I think this could generate too much log.
Could you put `logger.debug()` instead ?